spec/lib/bugzilla_service_spec.rb

```diff
@@ -1,5 +1,5 @@
 # rubocop:disable Style/NumericLiterals, Layout/IndentArray
-require 'spec_helper'
+require 'rails_helper'
```

Comment:
I think let's go back to spec_helper, but in addition, just drop the require 'spec_helper' from every file because it's not needed if spec_helper is set in the .rspec file.